common/views/components/IIIFPresentationPreview/IIIFPresentationPreview.js

```diff
@@ -101,13 +101,18 @@ const IIIFPresentationDisplay = ({
                   <a
                     href={iiifImageTemplate(
                       sequence.canvases[0].thumbnail.service['@id']
-                    )({ size: ',1024' })}
+                    )({ size: '!1024,1024' })}
                   >
                     <img
-                      style={{ width: 'auto', height: '300px' }}
+                      style={
+                        sequence.canvases[0].thumbnail.service.width <=
+                        sequence.canvases[0].thumbnail.service.height
+                          ? { width: 'auto', height: '300px' }
+                          : { width: '300px', height: 'auto' }
+                      }
```

Comment:
We should probably create a component that contains and image and constrains it to a bounding box.

This sort of does that:
https://github.com/wellcometrust/wellcomecollection.org/blob/master/catalogue/webapp/components/WorkCard/WorkCard.js#L161